.github/workflows/build.yml

```diff
@@ -49,9 +58,26 @@ jobs:
           free -m
           docker system df
           docker system df -v
-      - name: Build EVE KVM
+
+  eve:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        arch: [arm64, amd64]
+        hv: [xen, eve]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: update linuxkit cache if available
```

Comment:
In that case you will load linuxkit cache, which is good. But we still need some packages to be available in docker, as we discussed previously https://github.com/lf-edge/eve/pull/2727#issuecomment-1236695661. How to get them in place here?

Reply:
That is a very good point. I hadn't thought about that.

How does it work in `publish.yaml`, where it currently uses the remote runners to build and publish packages `make pkgs`, and then the local runner to `make eve`. Is it relying on the fact that it already had run `make pkgs` there?

Reply:
Correct, we did make pkgs before, so have all in docker. Also we publish packages firstly, so we can pull them from DockerHub.

Reply:
Probably [cache-export](https://github.com/lf-edge/eve/blob/e6c69cc972828a70e6e788c4cea02b4a40ce4f53/Makefile#L614-L616) may help us to extract images from linuxkit cache.

Reply:
We maintain the list of required docker packages in the `Makefile`:

```makefile
PKGS_DOCKER_LOAD=mkconf mkimage-iso-efi mkimage-raw-efi mkrootfs-ext4 mkrootfs-squash
```

We could do a `make load-docker-packages` which would export/import. I am worried it is a bit convoluted, but we can try it. What do you think, @giggsoff ?

Reply:
Agree, not so clear, but we can try at least.